### ISSUE TITLE
Navbar

### ISSUE THREAD

**Issue by bienvenu** [ocaml/v2.ocaml.org#279]
Once again, a pretty subjective issue:

Many people are annoyed by fixed elements on web sites. Top navbars have a special status regarding this, so for example although I hate fixed elements (they catch my eye whenever I scroll), I usually don't have any problems with them.

However, in the case of the new OCaml website, I think the navbar is slightly to big to be fixed. Indeed, I showed the new design to three friends this afternoon, and their reaction was unanimous: the first thing they said was that the navbar should not be fixed.

I have to say that I showed them the website simultaneously, so there might have been a bit of Asch-like conformity involved, but still.

Moreover, one of them had a pretty good point: he opened the website on his Ee-PC and indeed, the navbar covered a ridiculously large proportion of the screen on it.

So to recap, I think the navbar should be either slighly smaller (like on the old site) OR not fixed.

What do you think about this ?

**Comment by amirmc** [comment]
Thanks for the feedback. 

> he opened the website on his Ee-PC and indeed, the navbar covered a ridiculously large proportion of the screen on it

Would you be able to get a screenshot of this from your friend?  When I browse from my phone, the nav bar is not fixed and is similar to that of other sites I visit.  There may be an argument for narrowing the bar a little but in general I've found it helpful to have it fixed.  Of course, I haven't seen the site from a tablet (or similar resolution) so I'm not sure what the experience is like.

**Comment by bienvenu** [comment]
I had posted a reply 2 days ago and I've no idea why it's not showing here (it might be that I previewed it and then forgot to comment)

Anyway, here is the screenshot:

![screenshot_maxime_previewocaml](https://f.cloud.github.com/assets/4896692/1562027/2f93a5a0-503f-11e3-80df-9c6836111e6f.png)

As you can see it's about 1/6 of the page.

**Comment by Chris00** [comment]
@bienvenu May you tell me whether the above commit fixes the problem you report?  It should not affect the default style when the window height is larger than 600.

**Comment by bienvenu** [comment]
I don't have a Ee-PC / netbook, but here is a screenshot from my friend:

![screen_shot_maxime_2](https://f.cloud.github.com/assets/4896692/1577810/14513582-5170-11e3-98e3-89cb9bfc0d56.png)

I think its much better. I'll try to seek advice from other friends who have small screens, though.

**Comment by Chris00** [comment]
> I think its much better. I'll try to seek advice from other friends who have small screens, though.

OK.  It is not possible to make the banner smaller unless one uses a smaller logo.
